book/website/community-handbook/community-calls.md

```diff
@@ -0,0 +1,63 @@
+
+It is important to note that not all of our community calls fit neatly in one category over another. Some may mix practices and purposes (i.e. for coworking as well as celebrations), while others may have a specific purpose (i.e. for decision-making). All are meant to demonstrate our principles of being an open and transparent community.
+
+Because of their scope and size, the {ref}`Book Dash<ch-bookdash>` and {ref}`Fireside Chat series<ch-fireside-chat>` are documented separately. However, those calls overlap with many of the principles and practices shared in this community calls chapter.
+
+## 1. Coworking calls
+
+**Community calls**: {ref}`Weekly Coworking<ch-community-calls-coworking>`, {ref}`Collaboration Cafe<ch-community-calls-collabcafe>`
+
+The concept of coworking is generally associated with the practice of sharing physical office space with different people who work independently such as remote employees or freelancers.
+This allows people, who otherwise may feel isolated while working alone at home, to use traditional office setups, create a work routine, and build a sense of companionship with others who they share their office spaces with.
+
+We bring the practice of coworking to _The Turing Way_ using online coworking calls, through which we intend to develop a sense of energy and inspiration similar to in-person interactions but without the time and travel commitments.
+
+Our coworking calls provide opportunities for community members to allocate time to their desired tasks in _The Turing Way_ and get them done, partially or fully, with the support of others in the call who can help discuss their plans, share ideas or hold accountability.
+
+## 2. Organising and Governance Meetings
+
+**Community calls**: {ref}`Core Team Meetings (now retired)<ch-community-calls-core-team>`, {ref}`Open Governance Calls<ch-community-calls-governance-forum>`
+
+Community calls may also be used at spaces for organising and planning within _The Turing Way_ community for goverance purposes. As the community has grown tremendously (over 450 contributors and 30+ core team members as of February 2024), it has increasingly become clear that formalised decision-making and structures are needed to ensure the sustainability of the project.
+
+These calls may take on a more traditional meeting format, with a prepared agenda with particular points to be voted on by different members. Alternatively, it could also take on a more decentralised format that is aimed at gathering feedback from the organising body of the project, and or relay feedback from the Maintainer level to the Constitutional level of the project (see: [Liberating Structures](https://www.liberatingstructures.com/)). 
+
+In principle, these organising and governance meetings are still open for all to attend from the community as observers.
```

Comment:
I would add: 
Modelled off a political town hall event these online sessions are an opportunity to better understand activities across the project and how decisions are made. Our aim is to leverage the expertise across our community and we explicitly invite you to review our processes and recommend improvements. We expect that the calls will be a mix of celebration, feedback, and opportunities to build connections with other organisations and initiatives around the world.
Currently, this call is hosted in English. This call will be recorded for use by participants, and will use Zoom's automatic captioning. This call may also involve live discussion, screen-sharing, and other interactive activities that may be internet bandwidth-intensive. Slides will be shared after the call. A shared document will be used for collaborative note-taking, capturing feedback and learning.

Reply:
Thanks so much Alex! I'm saving this text here to add and edit this: These calls will provide an open forum for sharing  project updates, discussing governance-related work and inviting feedback & involvement from our community – join us!